src/pages/workspaces/workspace/Data.js

```diff
@@ -545,9 +552,13 @@ const EntitiesContent = ajaxCaller(class EntitiesContent extends Component {
                         ])
                       ]),
                       cellRenderer: ({ rowIndex }) => {
-                        return renderDataCell(
-                          Utils.entityAttributeText(entities[rowIndex].attributes[name]), namespace
-                        )
+                        const dataInfo = entities[rowIndex].attributes[name]
+                        return h(Fragment, [
```

Comment:
I'm not positive, but `h(Fragment, ...)` feels like unneeded wrapping. Is the result of `linkButton()` or `renderDataCell()` an acceptable return value for this function?